stores/GlobalPath/GlobalPathTypes.ts

```diff
@@ -0,0 +1,13 @@
+export type WayPoint = {
+    latitude: Number,
+    longitude: Number
+}
+
+export type WayPoints = {
```

Comment:
Sorry, I might have been unclear in our discussion. I meant to set `type WayPoints = WayPoint[]`, but, actually, this works better. Could you please change the name `WayPoints` to `GlobalPath`, just to make it clear that this corresponds to a single `GlobalPath` document?